applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_client.c

```diff
@@ -1089,6 +1272,15 @@ static void discover_cb(struct bt_conn *conn, int err, enum bt_audio_dir dir)
 		return;
 	}
 
+	/* If we have a valid stereo idx, put it on the queue */
+	if (memcmp(&stereo_idx, &idx, sizeof(struct stream_index))) {
```

Comment:
Not sure why a memcmp returning a non-0 value here indicates that something is valid?